.changeset/lazy-lamps-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-storybook": minor
```

Comment:
```suggestion
"@justeattakeaway/pie-storybook": minor
```

Please make sure this is commited before this PR is merged, otherwise `main` will fail, as the package now has the JET scope.

Reply:
This isn't being merged mate - its a proof of concept branch